docs/dev-guide/installingLocal.md

```diff
@@ -128,16 +96,43 @@ The previous shells were launched in their own containers. If you want to open a
   docker exec -it CONTAINER_NAME /bin/bash
   ```
 
-## Using tmux in a container
```

Comment:
I removed the tmux notes; I generally don't like prescribing specific development workflows.

Reply:
Agreed completely. You might consider getting rid of `contrib/start_workspace_tmux.sh`.

Reply:
Removed in 96310b4cba5a90a1ad6027b5aa9f997f979c2e42.